chainer/links/connection/mlp_convolution_2d.py

```diff
@@ -33,26 +36,44 @@ class MLPConvolution2D(link.ChainList):
         activation (function): Activation function for internal hidden units.
             Note that this function is not applied to the output of this link.
         conv_init: An initializer of weight matrices
-            passed to the convolution layers.
+            passed to the convolution layers. This option must be specified as
+            a keyword argument.
         bias_init: An initializer of bias vectors
-            passed to the convolution layers.
+            passed to the convolution layers. This option must be specified as
+            a keyword argument.
+
+    .. note:
+        From v2, `conv_init` and `bias_init` arguments must be specified as
+        keyword arguments only. We impose this restriction to forbid
+        users to assume the API for v1 and specify `wscale` option, 
+        that had been between `activation` and `conv_init` arguments in v1.
 
     See: `Network in Network <https://arxiv.org/abs/1312.4400v3>`_.
 
     Attributes:
         activation (function): Activation function.
 
-    """
+    """  # NOQA
 
     def __init__(self, in_channels, out_channels, ksize=None, stride=1, pad=0,
-                 activation=relu.relu, wscale=1, conv_init=None,
-                 bias_init=None):
+                 activation=relu.relu, *args, **kwargs):
+
+        # If `args` is not empty, users assume the API for v1 and
+        # specifie `wscale` as a positonal argument, which we want
+        # to detect and forbid with an explicit error message.
+        msg = ('wscale is not supported anymore. '
+               'Use conv_init and bias_init argument to change '
+               'the scale of initial parameters.')
+        if args:
```

Comment:
I am thinking of it. But I will do that after we need to do similar things in other functions.